mdbenchmark/utils.py

```diff
@@ -57,7 +59,7 @@ def get_possible_hosts():
 
 def print_possible_hosts():
     all_hosts = get_possible_hosts()
-    click.echo('Available host templates:')
+    formatting.info('Available host templates:')
     for h in all_hosts:
         click.echo('{}'.format(h))
```

Comment:
I let this in on purpose, as this should always be printed without any further formatting (in case `formatting.info` changes in the future.)